projects/valtimo/dashboard/src/lib/data-sources/task-count/components/task-count-configuration/task-count-configuration.component.html

```diff
@@ -0,0 +1,43 @@
+<!--
+  ~ Copyright 2015-2024 Ritense BV, the Netherlands.
+  ~
+  ~ Licensed under EUPL, Version 1.2 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" basis,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<form
+  class="task-count-configuration-form"
+  *ngIf="{operatorItems: operatorItems$} as obs"
+  [formGroup]="form"
+>
+  <div class="form__element conditions-multi-input">
+    <cds-label
+      [helperText]="'conditionsHelperText' | widgetTranslate: dataSourceKey | async"
+      class="conditions-label"
+      >{{ 'conditions' | widgetTranslate: dataSourceKey | async }}</cds-label
+    >
+    <valtimo-carbon-multi-input
+      [addRowText]="'addCondition' | widgetTranslate: dataSourceKey | async"
+      [defaultValues]="defaultConditionValues$ | async"
+      [dropdownColumnTitle]="'operator' | widgetTranslate: dataSourceKey | async"
+      [dropdownItems]="operatorItems$ | async"
+      [fullWidth]="true"
+      [initialAmountOfRows]="0"
+      [keyColumnTitle]="'path' | widgetTranslate: dataSourceKey | async"
+      [minimumAmountOfRows]="0"
+      [valueColumnTitle]="'value' | widgetTranslate: dataSourceKey | async"
+      type="keyDropdownValue"
+      (allValuesValidEvent)="onAllConditionsValid($event)"
+      (valueChange)="conditionsValueChange($event)"
+    ></valtimo-carbon-multi-input>
+  </div>
+</form>
```

Comment:
Can't the form just be the wrapper instead of the div?

Reply:
This is the way the framework is set up, there some styling already available for form__element. Would rather keep it this way, since this config form might be expanded in the future